dev/NavigationView/NavigationView_InteractionTests/NavigationViewTests.cs

```diff
@@ -742,7 +769,7 @@ public void VerifyNavigationViewItemResponseToClickAfterBeingMovedBetweenFrames(
                 var myLocationButton = FindElement.ByName<Button>("MyLocation");
                 var switchFrameButton = FindElement.ByName<Button>("SwitchFrame");
                 var result = new TextBlock(FindElement.ByName("MyLocationResult"));
- 
+
```

Comment:
Any reason format is changed a lot in this file?

Reply:
Oh I must have used auto format on that file and forgot to unstage the reformatted lines. Should I "revert" that?  :sweat_smile:

Reply:
Most of the reformatting looks fine to me. The lambda changing to multiple lines is a little weird but i'm fine with that too.